docker/nginx/conf.d/client.conf

```
@@ -1,6 +1,21 @@
 proxy_cache_path /data/nginx/cache levels=1:2 keys_zone=skynet:10m max_size=10g use_temp_path=off;
-limit_req_zone $binary_remote_addr zone=stats_by_ip:10m rate=10r/m;
-limit_conn_zone $binary_remote_addr zone=uploads_by_ip:10m;
+
+# ratelimit specified IPs
+geo $limit {
+	default 0;
+	include /etc/nginx/conf.d/include/ratelimited;
+}
+map $limit $limit_key {
+	0 "";
+	1 $binary_remote_addr;
+}
+
+limit_req_zone $binary_remote_addr zone=upload_req:10m rate=10r/s;
+limit_req_zone $limit_key zone=upload_req_rl:10m rate=10r/m;
```

I would suggest using `_throttled` suffix to the zone name instead of `_rl` - it is more descriptive than an abbreviation that is not expanded anywhere else. Also I think `uploads_by_ip` name is more descriptive than `upload_req` - I think we already changed that name following your advise in #148 

```suggestion
limit_req_zone $binary_remote_addr zone=uploads_by_ip:10m rate=10r/s;
limit_req_zone $limit_key zone=uploads_by_ip_throttled:10m rate=10r/m;
```